src/NzbDrone.Core/ImportLists/Simkl/SimklAPI.cs

```diff
@@ -75,6 +76,12 @@ public class SimklTvSyncActivityResource
     public enum SimklAnimeType
     {
         Tv,
-        Movie
```

Comment:
A value of `Unknown` above `Tv` would be best to capture all the types Sonarr doesn't know about, which will be used if it doesn't match a known type.

Reply:
My mistake, I forgot that parsing that enum wouldn't work with the unknown values out of the box, but what you found will work.